src/k8s-runtime/azext_k8s_runtime/tests/latest/test_k8s_runtime.py

```diff
@@ -0,0 +1,100 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+import os
+import tempfile
+import azext_k8s_runtime.custom_commands.storage_class as sc
+import azext_k8s_runtime.custom_commands.load_balancer as lb
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, live_only
+
+
+def _get_test_data_filepath():
+    with tempfile.NamedTemporaryFile(delete=False) as temp:
+        return temp.name
+
+
+CANARY_REGION = "eastus2euap"
+
+
+class K8sRuntimeScenario(ScenarioTest):
+    def create_connected_cluster(self, resource_group):
+        managed_cluster_name = self.create_random_name(prefix='test-connect', length=24)
+        kubeconfig = _get_test_data_filepath()
+        self.kwargs.update({
+            'rg': resource_group,
+            'name': self.create_random_name(prefix='cc-', length=12),
+            'managed_cluster_name': managed_cluster_name,
+            "kubeconfig": kubeconfig,
+            'location': CANARY_REGION,
+        })
+
+        self.cmd('aks create -g {rg} -n {managed_cluster_name} --generate-ssh-keys')
+        self.cmd('aks get-credentials -g {rg} -n {managed_cluster_name} -f \'{kubeconfig}\' --admin')
+        connected_cluster = self.cmd('connectedk8s connect -g {rg} -n {name} -l {location} --kube-config \'{kubeconfig}\' --kube-context {managed_cluster_name}-admin', checks=[
+            self.check('resourceGroup', '{rg}'),
+            self.check('name', '{name}')
+        ])
+
+        connected_cluster_resource_uri = connected_cluster.get_output_in_json()["id"]
+
+        self.kwargs.update({
+            "resource_uri": connected_cluster_resource_uri,
+        })
+
+    @live_only()
```

Comment:
Why do you need to mark these tests as `@live_only()` ?

Reply:
Actually I don't know why, but the recorded one doesn't work. I assume it has something to do with `az connectedk8s get-credentials` command, because the `connectedk8s` extension also uses `@live_only()` when a test called this command. See the tests here https://github.com/Azure/azure-cli-extensions/blob/b2c1d3e82027de4faceecaf79d508e91303b18f5/src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py#L148